src/Models/SolrLog.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+
+namespace Firesphere\SolrSearch\Models;
+
+use SilverStripe\Control\Director;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Security\Member;
+
+/**
+ * Class \Firesphere\SolrSearch\Models\SolrError
+ *
+ * @property string $Timestamp
+ * @property string $Message
+ * @property string $Index
+ * @property string $Type
+ * @property string $Level
+ */
+class SolrLog extends DataObject
```

Comment:
This is what I always wanted 👍 Dreams can come true!

Reply:
☁️